apps/agreement/contracts/test/mocks/disputable/AragonAppFeesCashierMock.sol

```diff
@@ -0,0 +1,61 @@
+pragma solidity ^0.4.24;
+
+import "@aragon/os/contracts/common/EtherTokenConstant.sol";
+import "@aragon/os/contracts/common/IsContract.sol";
+import "@aragon/os/contracts/lib/arbitration/IAragonAppFeesCashier.sol";
+
+
+contract AragonAppFeesCashierMock is IAragonAppFeesCashier, EtherTokenConstant, IsContract {
+    string private constant ERROR_WRONG_TOKEN = "SFO_WRONG_TOKEN";
```

Comment:
Just curious but what does `SFO` stand for?

Reply:
“San Francisco International Airport”? ;-P This was carried over from “Subscription Fees Oracle”, will fix it.